Code/src/main/java/nl/utwente/viskell/ui/components/BlockContainer.java

```diff
@@ -15,4 +17,25 @@
      */
     public void handleConnectionChanges(boolean finalPhase);
 
+    /** Gets the value of the property boundsInLocal. */
+    public Bounds getBoundsInLocal();
+    
+    /** Transforms a point from the local coordinate space of this Node into the coordinate space of its scene. */
+    public Bounds localToScene(Bounds bounds);
```

Comment:
Yes, however BlockContainer can't inherit from Node since that would force it to be a class instead of an interface. Luckily it does use the implementation of Node (or any override down to Lane/LambdaContainer) when requested.